Discussion/Discussion/Presentation/Comments/Base/ParentCommentView.swift

```diff
@@ -46,9 +46,8 @@ public struct ParentCommentView: View {
                 KFImage(URL(string: comments.authorAvatar))
                     .onFailureImage(KFCrossPlatformImage(systemName: "person"))
                     .resizable()
-                    .background(Color.gray)
                     .frame(width: 48, height: 48)
-                    .cornerRadius(isThread ? 8 : 24)
+                    .cornerRadius(24)
```

Comment:
is this change acceptable for upstream?
<img width="895" alt="Screenshot 2024-10-18 at 14 47 46" src="https://github.com/user-attachments/assets/0e829d59-1574-4c18-9a42-c5428d435989">

Reply:
I guess so because this is the only place where the user avatar isn't circular. So after this change, the user avatar will be consistent across the app.

Reply:
Hello @marcotuts and @sdaitzman. Could you please confirm that this almost square avatar is not a design feature and should be the same throughout the app? Thank you

Reply:
to me the avatar being square and circle is a design inconsistency. It should either be circle everywhere or square everywhere. For 2U we have a circular approach to icons and buttons.

Reply:
I agree with you. Just need confirmation we shouldn't add theming here.

Reply:
Will review and confirm asap - all being either square or circular makes sense, will just check existing app and mocks asap.